airflow/providers/amazon/aws/transfers/mysql_to_s3.py

```diff
@@ -54,12 +59,17 @@ class MySQLToS3Operator(BaseOperator):
                 You can specify this argument if you want to use a different
                 CA cert bundle than the one used by botocore.
     :type verify: bool or str
-    :param pd_csv_kwargs: arguments to include in pd.to_csv (header, index, columns...)
+    :param pd_csv_kwargs: deprecated. Use pd_kwargs instead.
+        Arguments to include in pd.to_csv (header, index, columns...)
```

Comment:
There is no need to change the docstring.
you just need to deprecate the param:

        if pd_csv_kwargs:
            warnings.warn(
                "pd_csv_kwargs is deprecated. Please use pd_kwargs.",
                DeprecationWarning,
                stacklevel=2,
            )
            self.pd_kwargs = pd_csv_kwargs